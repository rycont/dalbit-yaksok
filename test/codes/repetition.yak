횟수 = 0

반복
	횟수 보여주기
	횟수 = 횟수 + 1

	만약 횟수 == 10 이면
		반복 그만

5번 반복
	"A" 보여주기

반복 3번
	"B" 보여주기

목록 = ["X", "Y", "Z"]
반복 목록 의 항목 마다
	항목 보여주기

목록 = ["M", "N"]
목록 의 항목 마다 반복하기
	항목 보여주기

목록 = ["A", "B"]
2번 반복
	반복 목록 의 항목 마다
		항목 보여주기

숫자 = 0
10번 반복
	숫자 = 숫자 + 1
	숫자 보여주기
	만약 숫자 == 3 이면
		반복 그만
