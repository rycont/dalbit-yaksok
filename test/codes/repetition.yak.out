0
1
2
3
4
5
6
7
8
9
A
A
A
A
A
B
B
B
X
Y
Z
M
N
A
B
A
B
1
2
3
